pkg/ebpf/uprobes/testutil.go

```diff
@@ -153,3 +154,51 @@ func getLibSSLPath(t *testing.T) string {
 func (ua *UprobeAttacher) SetRegistry(registry FileRegistry) {
 	ua.fileRegistry = registry
 }
+
+// checkIfEventually checks a condition repeatedly until it returns true or a timeout is reached.
+func checkIfEventually(condition func() bool, checkInterval time.Duration, checkTimeout time.Duration) bool {
```

Comment:
How is this different from `assert/require.Eventually`?

Reply:
Both of them will fail the test if the condition isn't fulfilled at the end of the timeout, so they don't work for the function below where I want to retry the setup function (in this case, launching the process) in case of a failure.